busy_beaver/apps/github_summary/workflow.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import uuid
+from urllib.parse import urlencode
+
+from busy_beaver.config import GITHUB_CLIENT_ID, GITHUB_REDIRECT_URI, MEETUP_GROUP_NAME
+from busy_beaver.extensions import db
+from busy_beaver.models import User
+from busy_beaver.toolbox import EventEmitter
+
+#Setup logging
+logger = logging.getLogger(__name__)
+slash_command_dispatcher = EventEmitter()
+
+#String variables to describe state
+ACCOUNT_ALREADY_ASSOCIATED = (
+    "You have already associated a GitHub account with your Slack handle. "
+    "Please use `/busybeaver reconnect` to link to a different account."
+)
+NO_ASSOCIATED_ACCOUNT = (
+    "No associated account. Use `/busybeaver connect` to link your account."
+)
+VERIFY_ACCOUNT = (
+    "Follow the link below to validate your GitHub account. "
+    "I'll reference your GitHub username to track your public activity."
+)
+
+DELETE_ACCOUNT = (
+    "Account has been deleted. Run '/busybeaver connect' to reconnect"
+)
```

Comment:
Could you remove these constants from the other file?

Reply:
Resolved in this [commit](https://github.com/busy-beaver-dev/busy-beaver/pull/173/commits/9018690af331e34b56f9574d7bcc2bd243b9db28)